.github/workflows/release.yaml

```diff
@@ -1,280 +1,358 @@
 name: Release
 
+# Only trigger this workflow when a tag is pushed in the format `vA.B.C`.
 on:
   push:
     tags:
-      - '*'
-
+      - "v[0-9]+.[0-9]+.[0-9]+*"
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version - in the form of v1.2.3'
+        required: true
+        type: string
+
+# Configure constants for this workflow.
 env:
-  # update with the name of the main binary
-  binary: bevy_github_ci_template
-  add_binaries_to_github_release: true
-  #itch_target: <itch.io-username>/<game-name>
-
+  # The base filename of the executable produced by `cargo build`.
+  EXE_NAME: bevy_template
```

Comment:
I prefer the term binary instead of EXE because it's cross-platform.

Reply:
I suppose that makes sense. `EXE` is meant to be short for `EXECUTABLE` here, not `.exe`, but I see that that's ambiguous.

Reply:
Done